fastlane/Fastfile

```diff
@@ -2,32 +2,36 @@ default_platform(:android)
 fastlane_require 'dotenv'
 
 SUPPORTED_LOCALES = [
-  { glotpress: "ar", google_play: "ar" },
-  { glotpress: "de", google_play: "de-DE" },
-  { glotpress: "en-gb", google_play: "en-US" },
-  { glotpress: "es", google_play: "es-ES" },
-  { glotpress: "fr", google_play: "fr-CA" },
-  { glotpress: "fr", google_play: "fr-FR" },
-  { glotpress: "he", google_play: "iw-IL" },
-  { glotpress: "id", google_play: "id" },
-  { glotpress: "it", google_play: "it-IT" },
-  { glotpress: "ja", google_play: "ja-JP" },
-  { glotpress: "ko", google_play: "ko-KR" },
-  { glotpress: "nl", google_play: "nl-NL" },
-  { glotpress: "pl", google_play: "pl-PL" },
-  { glotpress: "pt-br", google_play: "pt-BR" },
-  { glotpress: "ru", google_play: "ru-RU" },
-  { glotpress: "sr", google_play:  "sr" },
-  { glotpress: "sv", google_play: "sv-SE" },
-  { glotpress: "th", google_play: "th" },
-  { glotpress: "tr", google_play: "tr-TR" },
-  { glotpress: "vi", google_play: "vi" },
-  { glotpress: "zh-cn", google_play: "zh-CN" },
-  { glotpress: "zh-tw", google_play: "zh-TW" },
+  { glotpress: "ar", google_play: "ar",  promo_config: {}},
+  { glotpress: "de", google_play: "de-DE",  promo_config: {} },
+  { glotpress: "en-gb", google_play: "en-US",  promo_config: {} },
+  { glotpress: "es", google_play: "es-ES",  promo_config: {} },
+  { glotpress: "fr", google_play: "fr-CA",  promo_config: {} },
+  { glotpress: "fr", google_play: "fr-FR",  promo_config: {} },
+  { glotpress: "he", google_play: "iw-IL",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
+  { glotpress: "id", google_play: "id",  promo_config: {} },
+  { glotpress: "it", google_play: "it-IT",  promo_config: {} },
+  { glotpress: "ja", google_play: "ja-JP",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
+  { glotpress: "ko", google_play: "ko-KR",  promo_config: {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60} },
```

Comment:
Why is it required to configure fonts for some locales but not others?

Reply:
There's a default font (Noto) defined in the plugin which is the one chosen by the designers for the screenshots. Unluckily, that font doesn't include a great support for not-latin characters, so we need to supply a different with better support for the locales that require them.

Reply:
Ah, that makes sense 👍